### PR TITLE
plugins/solidigm: Added Identify controller VU extensions parsing.

### PR DESCRIPTION
**Pull request by lgdacunh**

### CODE REVIEW

File: plugins/solidigm/solidigm-id-ctrl.c [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	uint8_t	rsvd1[3];
+	uint8_t ss;
+	char health[20];
+	uint8_t cls;
+	uint8_t nlw;
+	uint8_t scap;
+	uint8_t sstat;
+	char bl[8];
+	uint8_t rsvd2[38];
+	uint64_t ww; /* little endian */
```
Comment (igaw) [inline plugins/solidigm/solidigm-id-ctrl.c:23]: Couldn't you use `__le64` instead here?

File: plugins/solidigm/solidigm-id-ctrl.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ * Copyright (c) 2023 Solidigm.
+ *
+ * Author: leonardo.da.cunha@solidigm.com
+ */
+
+#include <inttypes.h>
+#include "common.h"
+#include "solidigm-id-ctrl.h"
+
+#pragma pack(push, vu_id_ctrl, 1)
```
Comment (igaw) [inline plugins/solidigm/solidigm-id-ctrl.c:12]: Why not using the attribute `__packed` ?

File: plugins/solidigm/solidigm-id-ctrl.c (hunk 1 above)
Comment (lgdacunh) [reply plugins/solidigm/solidigm-id-ctrl.c]: Done!

File: plugins/solidigm/solidigm-id-ctrl.c (hunk 2 above)
Comment (lgdacunh) [reply plugins/solidigm/solidigm-id-ctrl.c]: Done!

**Pull request merged**